<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ejercicio 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Elemento:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputElemento">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>341</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Lista destino</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="chA">
       <property name="text">
        <string>Lista A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="chB">
       <property name="text">
        <string>Lista B</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="agregar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listaA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listaB">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitarA">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitar de lista A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitarB">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitar de lista B</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
